truss/templates/custom/train/train.py

```diff
@@ -0,0 +1,18 @@
+from pathlib import Path
+
+
+class Train:
+    def __init__(
+        self,
+        config,
+        output_model_artifacts_dir: Path,
+        variables: dict,
+    ):
+        self.config = config
+        self.training_artifacts_dir = output_model_artifacts_dir
+        self.variables = variables
+
+    def train(self):
```

Comment:
yes, I think we need to add first class support for various model-types. I'm not sure yet how it will affect the API, but we'll modify it as needed to support that.